Extensions/popupLyrics.js

```diff
@@ -158,27 +188,41 @@ function PopupLyrics() {
 			}
 		}
 
+		/**
+		 * Search with PyNCM api.
+		 *
+		 * @param {Info} info
+		 * @returns {{
+		 * 	lyrics: [
+		 * 		{
+		 * 			startTime: Number,
+		 * 			text: string,
+		 * 		}
+		 * 	],
+		 * }}
+		 */
 		static async fetchNetease(info) {
-			const searchURL = `https://music.xianqiao.wang/neteaseapiv2/search?limit=10&type=1&keywords=`;
-			const lyricURL = `https://music.xianqiao.wang/neteaseapiv2/lyric?id=`;
-			const requestHeader = {
-				"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0"
-			};
+			const searchURL = `https://pyncmd.apis.imouto.in/api/pyncm?module=cloudsearch&method=GetSearchResult&keyword=`;
+			const lyricURL = `https://pyncmd.apis.imouto.in/api/pyncm?module=track&method=GetTrackLyrics&song_id=`;
```

Comment:
Yes, it is. However, the translation of the title of the album into Chinese was not handled separately because there was no translator in popupLyrics.

Reply:
huh. Can you apply the suggested change in `lyrics-plus/ProviderNeatease.js` as well

Reply:
What do you mean by suggested changes?